src/style.css

```diff
@@ -0,0 +1,32 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+
+  --star-width: 16px;
+  --star-height: 16px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background: url("images/star.svg") no-repeat center;
+  height: var(--star-height);
+  width: var(--star-width);
+}
+
+.stars--5 .stars__star,
+.stars--4 .stars__star:not(:last-child),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--1 .stars__star:first-child {
```

Comment:
Great usage of css selectors here :fire:

Reply:
Thx :blush: